doc/cla/individual/malaycmkhamar.md

```diff
@@ -0,0 +1,12 @@
+India, 2022-11-17
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Malay Khamar cmk.malay.khamar@gmail.com https://github.com/malaycmkhamar
```

Comment:
According to the [documentation](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#git)

Your commit message for the CLA should use the `[CLA]` tag, and I guess you should put your name instead of `Signing Odoo Individual CLA for odoo14`.

That is: 
```diff
- [ADD] Signing Odoo Individual CLA for odoo14
+ [CLA] Malay Khamar
```

Reply:
Ok @xavierbol ,
Done, Updated the CLA commit message.